velox/docs/functions/spark/datetime.rst

```diff
@@ -13,6 +13,14 @@ These functions support TIMESTAMP and DATE input types.
     If num_days is a negative value then these amount of days will be
     deducted from start_date.
 
+.. spark:function:: date_diff(endDate, startDate) -> integer
+
+    Returns the number of days from startDate to endDate.
```

Comment:
What are the times of startDate and endDate? Are these VARCHAR or DATE?

Reply:
Only DATE type is supported, just documented.